CHANGELOG.md

```diff
@@ -10,13 +10,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Added authority claimer service to support reader mode
+- Added `SFConfig`, `BHConfig` and `subscription_depth` configuration to `eth-input-reader`
 
 ### Changed
 
 - Renamed `dispatcher` service to `eth-input-reader`
 
 ### Removed
```

Comment:
Let's remove the state-server from the repository in this PR because it won't be used anymore.